integration/disable_test.go

```diff
@@ -23,21 +24,57 @@ func TestDisable(t *testing.T) {
 	defer cancel()
 	f.WaitForAllPodsReady(ctx, "app=disabletest")
 
-	setDisabled(f, "disabletest", true)
+	setDisabled(f.fixture, "disabletest", true)
 
 	f.WaitUntil(ctx, "pod gone", func() (string, error) {
 		out, err := f.runCommand("kubectl", "get", "pod", namespaceFlag, "-lapp=disabletest", "--no-headers")
 		return out.String(), err
 	}, "No resources found")
 
-	setDisabled(f, "disabletest", false)
+	setDisabled(f.fixture, "disabletest", false)
 
 	f.WaitForAllPodsReady(ctx, "app=disabletest")
 }
 
-func setDisabled(f *k8sFixture, resourceName string, isDisabled bool) {
-	out, err := f.runCommand(
-		"tilt",
+func TestDisableDC(t *testing.T) {
+	f := newDCFixture(t, "disable")
+	defer f.TearDown()
+
+	f.TiltUp("-f", "Tiltfile.dc")
+
+	ctx, cancel := context.WithTimeout(f.ctx, time.Minute)
+	defer cancel()
+
+	psArgs := []string{
+		"ps", "-f", "name=disabletest", "--format", "{{.Image}}",
+	}
+
+	f.WaitUntil(ctx, "service up", func() (string, error) {
+		return f.dockerCmdOutput(psArgs)
+	}, "disabletest")
+
+	f.WaitUntil(ctx, "disable configmap available", func() (string, error) {
+		cmd := exec.Command("tilt", "--port", fmt.Sprintf("%d", f.tilt.port), "get", "cm")
+		out, err := cmd.CombinedOutput()
+		return string(out), err
```

Comment:
FYI you can do `f.tilt.Get(ctx, "cm", "disabletest-disable")` (will give back JSON output as `[]byte` and err if any)

Reply:
nice